### PR TITLE
Support netstandard2.0

### PR DESCRIPTION
**Pull request by SzymonPobiega**

### CODE REVIEW

File: src/ServiceControl.TransportAdapter/FailedMessageForwarder.cs [hunk 1]
```diff
@@ -72,11 +72,11 @@ Task OnRetryMessage(MessageContext context)
 
         }
 
-        static Task Forward(Dictionary<string, string> newHeaders, MessageContext context, IDispatchMessages forwarder, string destination)
+        static async Task Forward(Dictionary<string, string> newHeaders, MessageContext context, IDispatchMessages forwarder, string destination)
```
Comment (bording) [inline src/ServiceControl.TransportAdapter/FailedMessageForwarder.cs]: @SzymonPobiega Why was this change made?

Comment (SzymonPobiega) [reply src/ServiceControl.TransportAdapter/FailedMessageForwarder.cs]: Crap. Only for debugging  and forgot to revert it. Good catch! ❤️

File: src/AcceptanceTests/AcceptanceTests/ConfigureEndpointLearningPersistence.cs [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+            tempDir = @"c:\temp";
+        }
+        else
+        {
+            tempDir = Path.GetTempPath();
+        }
+
+        storageDir = Path.Combine(tempDir, testRunId);
+
+        configuration.UsePersistence<InMemoryPersistence, StorageType.Subscriptions>();
```
Comment (andreasohlund) [inline src/AcceptanceTests/AcceptanceTests/ConfigureEndpointLearningPersistence.cs]: The learning transports doesn't need any storage for timeouts and subscriptions?

Comment (bording) [reply src/AcceptanceTests/AcceptanceTests/ConfigureEndpointLearningPersistence.cs]: @andreasohlund That's just a copy & past from core: https://github.com/Particular/NServiceBus/blob/develop/src/NServiceBus.AcceptanceTests/ConfigureEndpointLearningPersistence.cs

Comment (SzymonPobiega) [reply src/AcceptanceTests/AcceptanceTests/ConfigureEndpointLearningPersistence.cs]: I've removed the persistence config entirely.

Comment (andreasohlund) [reply src/AcceptanceTests/AcceptanceTests/ConfigureEndpointLearningPersistence.cs]: @bording turns out that the core needs those lines for the tests that use the FakeTransport eg. https://github.com/Particular/NServiceBus/blob/develop/src/NServiceBus.AcceptanceTests/Core/FakeTransport/CriticalError/When_registering_custom_critical_error_handler.cs#L39

So core needs them  but downstream should be ok to not use

**Pull request merged**